pallets/committee-management/src/impls.rs

```diff
@@ -423,6 +436,56 @@ impl<T: Config> Pallet<T> {
             Self::deposit_event(Event::BanValidators(fresh_bans));
         }
     }
+
+    /// Calculates committee for the given session.
+    /// If the era `E` starts in the session `a`, and ends in session `b` then from session `a-1`
+    /// to session `b-1` this function can answer question who will be in the committee in the era `E`.
+    /// In the last session of the era `E` this can be used to determine all of the sessions in the
+    /// era `E+1`.
+    pub fn session_committee_for_session(
+        session: SessionIndex,
+    ) -> Result<SessionCommittee<T::AccountId>, SessionValidatorError> {
+        let ce = match T::EraInfoProvider::current_era() {
+            Some(ce) => ce,
+            _ => return Err(SessionValidatorError::Other("No current era".encode())),
+        };
+
+        let current_starting_index = match T::EraInfoProvider::era_start_session_index(ce) {
+            Some(csi) => csi,
+            // Shouldn't happen
+            None => {
+                return Err(SessionValidatorError::Other(
+                    "No known starting session for current era".encode(),
+                ))
+            }
+        };
+
+        // Historical session
+        if session < current_starting_index {
+            return Err(SessionValidatorError::OldEra);
```

Comment:
Mayba it would be better for the user to combine this error with `SessionValidatorError::SessionTooFarIntoFuture` and output something like `SessionValidatorError::SessionNotWithinRange { lower_limit: current_starting_index, upper_limit: planned_era_end}`

Makes sense?

Reply:
Sure, done